recipe/meta.yaml

```
@@ -1,5 +1,5 @@
 {% set name = "cuda-python" %}
-{% set version = "11.8.1" %}
+{% set version = "11.8.2" %}
 {% set major_version = version.split(".")[0] %}
 {% set minor_version = version.split(".")[1] %}
 {% set patch_version = version.split(".")[2] %}
```

Suggesting adding Jinja filters to convert these to integers to enable comparisons

```suggestion
{% set major_version = version.split(".")[0]|int %}
{% set minor_version = version.split(".")[1]|int %}
{% set patch_version = version.split(".")[2]|int %}
```